### PR TITLE
replaced whitelist with allowlist

### LINKED ISSUE #41754: Replace blacklist/whitelist in tools/code_analyzer/gen_op_registration_whitelist.py
The goal of this issue is to replace all instances of *whitelist* and *blacklist* in `tools/code_analyzer/gen_op_registration_whitelist.py`. See issue #41443 for more information.

### PR DESCRIPTION
**Pull request by Vishal487**
Fixes #41754

**(1)**
Intially file was named **gen_op_registration_whitelist.py** I changed it to **gen_op_registration_allowlist.py**

**(2)**
There were some **whitelist** in comment inside the file, I changed it to **allowlist**
update1

### CODE REVIEW

**Comment by Vishal487** [comment]
@SplitInfinity Please review it.

File: docs/source/tensors.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ torch.Tensor
 A :class:`torch.Tensor` is a multi-dimensional matrix containing elements of
```
Comment (SplitInfinity) [inline docs/source/tensors.rst:8]: Please undo the changes to this file. They are not part of task.

**Review by SplitInfinity — CHANGES_REQUESTED** [review]
Thanks for working on this. Please undo the changes to one file that are not part of the task.

**Review by ljk53 — APPROVED** [review]

File: docs/source/tensors.rst (hunk 1 above)
Comment (ljk53) [reply docs/source/tensors.rst:8]: no worries, I've reverted this file when importing to Phabricator (I need change an internal BUCK reference to the renamed file, too).

**Comment by Vishal487** [comment]
Since I'm new here in open source world, so I don't know what to do next?
It seems PR is merged but I'm getting message that says **closed commit with unmerged commits**. Could anyone help me?

**Comment by SplitInfinity** [comment]
There is nothing left to do. This repository works slightly different than others in the sense that the source of truth is an internal Facebook repository to which commits are merged, and then an automated job makes those same commits in the GitHub repository. That's why your PR with closed without being merged.
